### ISSUE TITLE
Please don't change public API names

### ISSUE THREAD

**Issue by VBAndCs** [paul1956/CSharpToVB#80]
In this class I has a Value property:
```C#
public class Placeholder<T>
{
    T value;

    public T Value
    {
        get => value;
        set
        {
            Found = true;
            this.value = value;
        }
    }

    public bool Found = false;

    public override string ToString()
    {
        if (Found)
            return value?.ToString();

        return "Not Found";
    }
}
```

The converter changed it's name to Value1:

```vb
' To configure or remove Option's included in result, go to Options/Advanced Options...
Option Compare Text
Option Explicit On
Option Infer Off
Option Strict On

Public Class Placeholder(Of T)
    Private value As T
    Public Property Value1 As T
        Get
            Return value
        End Get

        Set(Value As T)
            Found = True
            Me.value = value
        End Set
    End Property
    Public Found As Boolean = False
    Public Overrides Function ToString() As String
        If Found Then
            Return value?.ToString()
        End If

        Return "Not Found"
    End Function
End Class
```

I know you are trying to avoid confection with the value  filed, but you can safely change the private field name. Changing the public property name Value to Value1 causes many errors, some of them can happen in reflection in frameworks that use naming convention. So, whenever the case conflict happens, change the private name not the public name.
Thanks.

**Comment by paul1956** [comment]
Converter 5.0.2.5 fixes part of this. I no longer rename Value to Value1 but value is not renamed either. I do add Me. to value

**Comment by VBAndCs** [comment]
> I no longer rename Value to Value1 but value is not renamed either

VB can't accept value and Value as two differenr names. I recommend to rename value to _value, but be aware not to change Value param in the set blocks in properties.

**Comment by paul1956** [comment]
@VBAndCs  I think I have a complete fix in 5.0.2.6. posted tomorrow. If not please provide complete source of C# and VB where conversion does not work.
